NOTICE.md

```diff
@@ -0,0 +1,75 @@
+# Notices for the libcoap-rs Project
+
+The libcoap-rs project (which includes both the libcoap-rs and libcoap-sys
+crates) is maintained by the (former) NAMIB project members, see [below](#maintainers)
+for an up-to-date list of people currently maintaining this repository.
+
+- [Project website: https://namib-project.github.io/](https://namib-project.github.io/)
+- [GitHub organization: https://github.com/namib-project/](https://github.com/namib-project/)
+- [libcoap-rs repository: https://github.com/namib-project/libcoap-rs](https://github.com/namib-project/libcoap-rs)
+
+## Maintainers
+
+This repository is currently maintained by the following developers:
+
+|       Name       |    Email Address     |               GitHub Username                |
+|:----------------:|:--------------------:|:--------------------------------------------:|
+| Hugo Hakim Damer | hdamer@uni-bremen.de | [@pulsastrix](https://github.com/pulsastrix) |
+
+## Reporting Security Vulnerabilities
+
+Security vulnerabilities may be reported using the
+[GitHub Vulnerability Reporting Tool](https://github.com/namib-project/libcoap-rs/security).
+If you prefer email, you may also report security vulnerabilities to any of the maintainers' email
+addresses listed above (ideally encrypted using PGP).
+*DO NOT* open a public GitHub issue for security vulnerabilities.
+
+When reporting a security vulnerability, please provide instructions on how to reproduce the issue.
+Do not send reports that were generated with automated vulnerability scanning or AI tools without
+verifying that they are not false positives or without providing additional context.
+
+Also, please ensure that reported security vulnerabilities pertain to libcoap-rs and/or libcoap-sys
+in particular, not to the libcoap C library or any libraries libcoap depends on.
+For instructions on reporting security vulnerabilities that pertain to libcoap, refer to
+[its own security policy](https://github.com/obgm/libcoap/blob/develop/SECURITY.md).
+
+As libcoap-rs is not maintained by a for-profit entity, we do not offer any monetary compensation
+for vulnerability or bug reports, but your contributions are greatly appreciated.
+
+Lastly, please note that as an open source project, libcoap-rs and libcoap-sys are provided "as is",
+i.e., without any warranty or guarantee of fitness for a particular purpose ([see below](#copyright-information)).
+
+### Security Vulnerability Hall of Fame
+
+We are very thankful to the following people for reporting security issues in the past:
+
+- None yet.
```

Comment:
Hmm, I am wondering if this could also be turned or partially moved into a `SECURITY.md` file for this repository.